packages/js/product-editor/src/blocks/generic/subsection/edit.tsx

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
There's a lot of copied code from the section block. I know it's on purpose, but at the same time, I wonder if we couldn't create a React component to avoid all that duplication. Even if we change the subsection design, having a component could come in handy, especially if the changes are not so drastic. What do you think?

Reply:
Hi @nathanss,
Yeah, I think that it's a good idea to refactor that. Since this PR is already quite big, what do you think about moving forward with this PR and I can tackle the refactor in a follow-up PR?

Reply:
Sure. Can you please create the follow-up issue?

Thanks!

Reply:
Great! Yes, I this is the follow-up issue https://github.com/woocommerce/woocommerce/issues/44476